### DISCUSSION TITLE
OBC add third mode for t to toggle headshot-stats-off

### DISCUSSION THREAD

**Discussion by JJWatMyself** [f1multiviewer/issue-tracker#176] [Feature requests] [answered]
OBC add third mode for t to toggle headshot-stats-off

Stats has impact on GPU, and toggle off addresses, but to have option to have headshot, when Live Stats window is open, is another option that will have low GPU overhead, whilst still identifying who the driver it.

**Comment by f1multiviewer** [comment, marked as answer]
This is available now!
